### ISSUE TITLE
`@api.validate` seems to not work with starletee class endpoint

### ISSUE THREAD

**Issue by deepalikumar** [0b01001001/spectree#25]
Hi.

I have this starlette endpoint.
```
class UserPostListEndpoint(HTTPEndpoint):
    @api.validate(resp=Response(HTTP_200=PostListSchema))
    async def get(self, request):
        user_id = request.path_params["user_id"]
        return UJSONResponse(await get_user_posts(user_id))

```

This endpoint is mounted as:

```
    routes = [
        Mount(
            "/api/v1",
            name="v1",
            routes=[
                Route(
                    "/users/{user_id:int}/posts", UserPostListEndpoint, name="user_list"
                )
            ],
        )
    ]
```

Invoking it throws an exception.
```
, in run
    result = self.fn(*self.args, **self.kwargs)
TypeError: validate() missing 2 required positional arguments: 'receive' and 'send'
```

Does this not work with starlette endpoint?

**Comment by kemingy** [comment]
Currently, this library doesn't support starlette endpoint. This should be a new feature.

**Comment by kemingy** [comment]
@deepalikumar Hi, this should be fixed in the latest version(0.2.7).
